src/content/guide/tools-and-features/remote-diagnostics.md

```diff
@@ -0,0 +1,320 @@
+---
+word: Remote Diagnostics
+title: Remote Diagnostics
+order: 11
+columns: two
+layout: guide.hbs
+devices: [photon, electron]
+---
+
+# {{title}}
+
+<p class="boxedHead">Beta feature</p>
+<p class="boxed">
+This feature is **currently in private beta**.
+If you would like access to this feature, please
+<a href="https://www.particle.io/sales" target="_blank">contact us</a>.
+</p>
+
+As you  deploy your IoT fleet into the field, it becomes increasingly
+important to ensure that devices stay in a healthy state. In addition,
+when problems do arise, the ability to quickly identify and implement a
+solution are paramount to the reliability of your production deployment.
+This level of visibility can often be difficult to achieve without
+physical access to these devices in the field.
+
+Remote Diagnostics gives your team the power to actively monitor the health of
+deployed units from the Console, without the need for custom development
+or costly dispatching of technicians. Furthermore, in the event of a
+device falling into an unhealthy state, your team will be empowered with
+rich context and suggested courses of action to quickly diagnose and
+rapidly resolve the issue.
+
+{{#if electron}}
+<img
+src="/assets/images/remote-diagnostics/successful-diagnostics-test.png"
+class="full-width"/>
+{{else}}
+<img
+src="/assets/images/remote-diagnostics/successful-diagnostics-test-wifi.png"
+class="full-width"/>
+{{/if}}
+<p class="caption">Remote Diagnostics allow you to test the connectivity
+health of your devices, and quickly resolve problems when they arise.</p>
+
+## Connectivity Layers
+
+Multiple connectivity layers must be operating successfully for a given device
+to be able to successfully communicate with the cloud. Note that the
+relevant connectivity layers vary based on the type of device (i.e.
+Wi-Fi vs. Cellular).
+
+These connectivity layers are:
+
+{{#if electron}}
+  <img class="full-width" alt="Device, SIM Card, Cellular Network, and
+  Particle Cloud"
+  src="/assets/images/remote-diagnostics/connectivity-layers-cellular.png"/>
+{{else}}
+  <img alt="Device and
+  Particle Cloud"
+  src="/assets/images/remote-diagnostics/connectivity-layers-wifi.png"/>
+{{/if}}
+
+
+### Device
+The device itself must be in a healthy state in order to successfully
+communicate with the cloud. A variety of factors influence its state,
+such as battery state of charge, signal strength, available memory, and
+application firmware that does not exceed enforced rate limits.
+
+{{#if electron}}
+### SIM Card
+Cellular devices rely on a SIM card to facilitate a connection to the
+cellular network. This SIM must be in an active state, allowing the
+device to try to initiate a data session with the network.
+
+### Cellular Network
+
```

Comment:
Extra empty row

Reply:
It looks like most of them have this empty row, but above, "Device" and "SIM Card" don't have this extra row

Reply:
Added line breaks for visibility